src/jobs.ts

```diff
@@ -0,0 +1,61 @@
+import { scheduleJob } from "node-schedule";
+import axios from "axios";
+import { redis } from "./index.js";
+import { syncServers, storeMessage } from "./util.js";
+import { Server, Message } from "./types.js";
```

Comment:
if these are all TS things, you can leave off the file extension and it'll compile with the right extension

Reply:
which requires one rule to be changed which requires more rules to be changed and then I think it eventually needs an entire different ES version

Reply:
No, you can take the file names off and it works just fine. I've used it before in ts projects.

Reply:
```suggestion
import { syncServers, storeMessage } from "./util";
import { Server, Message } from "./types";
```